sql/api/src/main/scala/org/apache/spark/sql/util/ArrowUtils.scala

```diff
@@ -219,4 +219,18 @@ private[sql] object ArrowUtils {
         valueContainsNull)
     case _ => dt
   }
+
+  def toArrowOutputSchema(dt: DataType): DataType = {
```

Comment:
nit: Since it focuses on handling UDT only, shall we either rename the function or add a comment to clarify its purpose?

Reply:
DOne